.github/workflows/build-images.yaml

```diff
@@ -219,6 +223,42 @@ jobs:
           build-args: |
             CA_GIT_TAG=${{ steps.get-ca-tag.outputs.tag }}
 
+      - name: Login to dev ECR
+        if: ${{ format('{0}', inputs.upload-to-ecr) == 'true' }}
+        uses: docker/login-action@v3
+        with:
+          registry: 369495373322.dkr.ecr.eu-central-1.amazonaws.com
+          username: ${{ secrets.DEV_GHA_RUNNER_LIMITED_AWS_ACCESS_KEY_ID }}
+          password: ${{ secrets.DEV_GHA_RUNNER_LIMITED_AWS_SECRET_ACCESS_KEY }}
+
+      - name: Login to prod ECR
+        if: ${{ format('{0}', inputs.upload-to-ecr) == 'true' }}
+        uses: docker/login-action@v3
+        with:
+          registry: 093970136003.dkr.ecr.eu-central-1.amazonaws.com
+          username: ${{ secrets.PROD_GHA_RUNNER_LIMITED_AWS_ACCESS_KEY_ID }}
+          password: ${{ secrets.PROD_GHA_RUNNER_LIMITED_AWS_SECRET_ACCESS_KEY }}
```

Comment:
Let's do this at the top, after the initial docker login? (unless we think it's more reliable to delay login, because at least we'd push to dockerhub if login to ECR isn't working)

Related: _may_ also want to consider whether we should "build and push" each image or just build locally and _then_ push everything in one go (allowing us to push to other registries if dockerhub is failing...)

Incremental benefit of that change is probably small though; happy to leave it for a follow-up.